openfecwebapp/templates/partials/committee-totals-house-senate.html

```diff
@@ -30,7 +30,6 @@
      (totals.0.refunded_individual_contributions, 'Individual refunds'),
      (totals.0.refunded_political_party_committee_contributions, 'Political party refunds'),
      (totals.0.refunded_other_political_committee_contributions, 'Other committee refunds'),
-     (totals.0.other_disbursements, 'Other disbursements'),
```

Comment:
So, we show "total disbursements" above this table in the header. If we do want it here, we should also add "total receipts" too.